### PR TITLE
Optional stuff

### PR DESCRIPTION
**Pull request by g9militantsYT**
- 1st commit removed an empty space skiddi boi added accidentally ig (I have an OCD)
- 2nd commit added a typing animation to the "Our pricing plan to disrupt the Market:" so it catches the user's attention 
- 3rd commit changed the text from "Our pricing plan to disrupt the Market:" to "Our pricing plans to disrupt the Market:" (change is plural 'plans' since multiple plans are now available)
- As a sidenote, never expect useful commits from me. Just, let's just say, quality updates 💀

### CODE REVIEW

File: pages/music-landing/main.ts [hunk 1]
```diff
@@ -364,3 +365,26 @@ Body(
         FullWidthSection(Footer()),
     ).setMaxWidth("1230px"),
 );
+
+// Typing Animation
+const typingTextElement = document.getElementById('typing-text');
```
Comment (lucsoft) [inline pages/music-landing/main.ts:370]: Can you do this in an A11y way? 💀💀

**Comment by GregTCLTK** [comment]
we don't need a 2013-vibe typing animation

**Review by lucsoft — CHANGES_REQUESTED** [review]

**Comment by g9militantsYT** [comment]
dang it I thought the typing animation looked cool 😢

**Comment by GregTCLTK** [comment]
if you'd like to contribute, you can add skeleton loaders or toast notifications

**Pull request closed without merging**